.gitignore

```diff
@@ -0,0 +1 @@
+Atest.md
```

Comment:
- [ ] As written in the project requirements. KIndly add `test.md` not `Atest.md` to your `.gitignore` file.
- [ ] Kindly also add `node_modules` directory to your `.gitignore file.